charts/pega/KafkaClusterRequirement.md

```diff
@@ -42,13 +42,21 @@ You may also securely pass settings like trustStorePassword,keyStorePassword, an
 #### Permissions
 To configure an externalized Kafka service connection using authentication and authorization profiles in Kafka Access control lists, your Pega profiles require following user permissions. To review configuration details, see [Kafka documentation for Authorization and ACLs](https://kafka.apache.org/documentation/#security_authz).
 
-Principal |Resource Type  | Resource Name     | Operation | Permission Type | Patter Type
+Principal |Resource Type  | Resource Name     | Operation | Permission Type | Pattern Type
 ---         |---         | ---     | ---    | ---        | ---
 User:\<user-name\> | TOPIC       | \<Prefix\> as in 'stream.streamNamePattern' | ALL    | ALLOW      | PREFIXED
 User:\<user-name\> |TRANSACTIONAL_ID  | * | READ/WRITE   | ALLOW      | LITERAL
 User:\<user-name\> |GROUP  | \<Prefix\> as in 'stream.streamNamePattern' | ALL   | ALLOW      | PREFIXED
 User:\<user-name\> |CLUSTER  | \<Cluster-Name\> | IDEMPOTENT_WRITE   |ALLOW      | LITERAL
 
+For e.g.
+```
+# To grant permissions to allow Delete, Create, Write, and Read operations on a topic based on a prefix in the 'stream.streamNamePattern', here prefix is 'pega'.
```

Comment:
I think this is helpful. thanks for thinking about this more...would this be more clear?
To grant permissions to allow Delete, Create, Write, and Read operations on a topic in the 'stream.streamNamePattern' which includes the prefix `pega'.

To grant permissions to allow describe and read operations on a group type resource in the 'stream.streamNamePattern' identified by the prefix `pega'.

Reply:
@taz-pega-work we do not specify topic names or group type in 'stream.streamNamePattern.
An e.g. is "pega-{stream.name}" here prefix is 'pega' and {stream.name} is the topic name/group name which is replaced at run time.
What about these statements?
To grant permissions to allow Delete, Create, Write, and Read operations on resource type TOPIC when the value of 'stream.streamNamePattern' is "pega-{stream.name}"
To grant permissions to allow describe and read operations on a resource type GROUP when the value of 'stream.streamNamePattern' is "pega-{stream.name}"

Reply:
@taz-pega-work please review once

Reply:
@taz-pega-work did you get a chance to review?

Reply:
looks good! sorry for the delay! Taz

Thanks for improving the sentence ;-)